src/classes/user-config-utils.ts

```diff
@@ -0,0 +1,54 @@
+import fs from "fs";
+import { UserConfigJson } from "../interfaces/types";
+import path from "path";
+
+const userConfigPath = `.${path.sep}user-config.json`;
+const userConfigTemplate = `.${path.sep}resources${path.sep}user-config-template.json`;
+
+export class userConfigUtils {
```

Comment:
Class names should be captitalized
```suggestion
export class UserConfigUtils {
```